File: blogmanager/blogmanagerweb/src/main/webapp/js/service/messageService.js
messageApp.service('messageService',function($http){
    this.findPage=function(page,rows){
        return $http.get('../message/list.do?page='+page+"&rows="+rows);
    }

    this.findOne=function(messageId){
        return $http.get('../message/findOne.do?messageId='+messageId);
    }

    this.addReply=function(entity){
        return $http.post('/message/addReply.do',entity);
    }
});

File: blogmanager/blogmanagerservice/src/main/java/com/blog/service/MessageService.java
package com.blog.service;

import com.blog.common.pojo.EUDataGridResult;
import com.blog.common.pojo.Result;
import com.blog.pojo.TbMessage;
import com.blog.pojo.TbMessageCustom;

public interface MessageService {
    EUDataGridResult getMessageList(Integer page, Integer rows);
    Result findOneMessage(Long messageId);
    Result addReply(TbMessageCustom messageCustom);
}
